styles/components/App.less

```diff
@@ -30,12 +29,12 @@
   margin-left: auto;
   margin-right: auto;
   margin-bottom: 20px;
-  min-height: 100px;
-  max-height: 420px;
+  min-height: 470px;
```

Comment:
By screens, do you mean displays, like MacBook Air 13" vs 11"? If so, we should consider making the window small by default and allow people to resize it vertically. 

Also, there's a lot of wasted space in the Uploader right now. For example, the T icon mark is huge and could be much smaller. Likewise, it seems odd to include both the T icon and the Tidepool logo, you could remove the Tidepool logo and make the footer much smaller. But again, what's implemented in based on the approved design, so I don't want to overstep.